integrations-and-supported-tools/colab/Neptune_Colab.ipynb

```diff
@@ -341,86 +341,85 @@
+    "* `file.txt` which will be visible under All Metadata/artifacts as sample.txt"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "5otsPw48mMaj"
+    "id": "XSHLD6AkI7LW"
    },
    "source": [
-    "**Tip:**<br>\n",
-    "To view the structure of a run, use the `print_structure()` method."
+    "# Stop logging  \n",
+    "<font color=red>**Warning:**</font><br>\n",
+    "Once you are done logging, you should stop tracking the run using the `stop()` method.\n",
+    "This is needed only while logging from a notebook environment. While logging through a script, Neptune automatically stops tracking once the script has completed execution."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "_r0Er7EHHram"
+    "id": "Zgh3NwDoJAuG"
    },
    "outputs": [],
    "source": [
-    "run.print_structure()"
+    "run.stop()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "XSHLD6AkI7LW"
+    "id": "mRhOAcS0Q7Mm",
+    "tags": [
+     "comment"
+    ]
    },
    "source": [
-    "# Step 4: Stop logging"
+    "# Explore the run in the Neptune UI\n",
+    "\n",
+    "![upload_file_as_artifact.png](https://gblobscdn.gitbook.com/assets%2F-MXTao65xpmg6bawJ5vk%2F-MYZt9HJXRALUZWcKCSo%2F-MY_5xqm2_MtE0usG2bV%2Fupload_file_as_artifact.png?alt=media&token=d861bd06-6389-475f-99fb-b34ec4bc3d06)\n",
+    "\n",
+    "Go to the `All metadata` and `Charts` sections of the Neptune UI to see them. You can also check an [example run](https://app.neptune.ai/o/common/org/showroom/e/SHOW-37/charts) \n",
+    "\n",
+    "![colab_example_charts.png](https://gblobscdn.gitbook.com/assets%2F-MXTao65xpmg6bawJ5vk%2F-MYZt9HJXRALUZWcKCSo%2F-MY_69KYOwzBEXczhIss%2Fcolab_example_charts.png?alt=media&token=3ffdc950-581d-4d44-8de3-bce0d4493f5c)\n",
+    "\n",
+    "**Tip:**\n",
+    "\n",
+    "Neptune automatically logs the hardware consumption during the run. \n",
+    "\n",
+    "You can see it in the `Monitoring` section of the Neptune UI. \n",
+    "\n",
+    "![image](https://neptune.ai/wp-content/uploads/Product_hardware-usage-1.png)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "IWuABAl_MBDT"
+    "id": "5otsPw48mMaj"
    },
    "source": [
-    "<font color=red>**Warning:**</font><br>\n",
-    "Once you are done logging, you should stop tracking the run using the `stop()` method.\n",
-    "This is needed only while logging from a notebook environment. While logging through a script, Neptune automatically stops tracking once the script has completed execution."
+    "**Tip:**<br>\n",
+    "To view the structure of a run, use the `print_structure()` method."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "Zgh3NwDoJAuG"
+    "id": "_r0Er7EHHram"
    },
    "outputs": [],
    "source": [
-    "run.stop()"
+    "run.print_structure()"
```

Comment:
You cannot call this on the closed run. It should probably be above the `run.stop()`.

Reply:
This will cause tests to fail.

Reply:
fixed